zips/zip-2001.rst

```diff
@@ -84,15 +84,97 @@ it will be modified to read:
+
+    - The coinbase transaction in each block MUST contain at least one output per
+      active funding stream that pays the stream's value in the prescribed way to
+      the stream's recipient address for the block's height.
+
+it will be modified to read:
+
+    - In each block, for each active funding stream with a recipient other than
+      `DEFERRED_POOL` at that block's height, the block's coinbase transaction
+      MUST contain at least one output that pays the stream's value in the
+      prescribed way to that recipient.
+
+After the list of post-Canopy consensus rules, the following paragraph is added:
+
+    The effect of the definition of :math:`\mathsf{PoolValue}_{Deferred}` above
+    is that payments to the `DEFERRED_POOL` cause
+    :math:`\mathsf{FundingStream[FUND].Value}(\mathsf{height})` to be added to
+    :math:`\mathsf{PoolValue}_{Deferred}` for the block chain including that block.
+
+
+Modifications to the protocol specification
+-------------------------------------------
+
+In section **7.1.2 Transaction Consensus Rules** [#protocol-txnconsensus]_, instead of:
+
+    The total value in zatoshi of transparent outputs from a coinbase transaction,
+    minus :math:`\mathsf{v^{balanceSapling}}`, minus :math:`\mathsf{v^{balanceOrchard}}`, MUST NOT
+    be greater than the value in zatoshi of the block subsidy plus the transaction
+    fees paid by transactions in this block.
+
+it will be modified to read:
+
+    For the block at height :math:`\mathsf{height}`, the total value in zatoshi of
+    transparent outputs from its coinbase transaction, minus :math:`\mathsf{v^{balanceSapling}}`,
+    minus :math:`\mathsf{v^{balanceOrchard}}`, plus :math:`\mathsf{totalDeferredOutput}(\mathsf{height})`,
+    MUST NOT be greater than the value in zatoshi of block subsidy plus the transaction
+    fees paid by transactions in this block.
+
+where :math:`\mathsf{totalDeferredOutput}(\mathsf{height})` is defined consistently
+with ZIP 207.
+
+Note: this ZIP and ZIP 236 both make changes to the above rule. Their combined effect
+is that it will read as follows:
+
+    [Pre-NU6] For the block at height :math:`\mathsf{height}`, the total value
+    in zatoshi of transparent outputs from its coinbase transaction, minus
+    :math:`\mathsf{v^{balanceSapling}}`, minus :math:`\mathsf{v^{balanceOrchard}}`, plus
+    :math:`\mathsf{totalDeferredOutput}(\mathsf{height})`, MUST NOT be greater
+    than the value in zatoshi of block subsidy plus the transaction fees paid
+    by transactions in this block.
+
+    [NU6 onward] For the block at height :math:`\mathsf{height}`, the total value
+    in zatoshi of transparent outputs from its coinbase transaction, minus
+    :math:`\mathsf{v^{balanceSapling}}`, minus :math:`\mathsf{v^{balanceOrchard}}`, plus
+    :math:`\mathsf{totalDeferredOutput}(\mathsf{height})`, MUST be equal to the
+    value in zatoshi of block subsidy plus the transaction fees paid by transactions
+    in this block.
```

Comment:
It might be clearer to express it like this:

> For the block at height :math:`\mathsf{height}`:
> * define the "total output" of its coinbase transaction to be the total value in zatoshi of its transparent outputs, minus :math:`\mathsf{v^{balanceSapling}}`, minus :math:`\mathsf{v^{balanceOrchard}}`, plus :math:`\mathsf{totalDeferredOutput}(\mathsf{height})`;
> * define the "total input" of its coinbase transaction to be the value in zatoshi of the block subsidy, plus the transaction fees paid by transactions in the block.
>
> [Pre-NU6] The total output of a coinbase transaction MUST NOT be greater than its total input.
>
> [NU6 onward] The total output of a coinbase transaction MUST be equal to its total input.